docs/src/SelectList.doc.js

```diff
@@ -20,185 +41,361 @@ card(
       {
         name: 'disabled',
         type: 'boolean',
+        description: `Used to disable the entire SelectList. See the [states](#States) variant to learn more.`,
         defaultValue: 'false',
       },
       {
         name: 'errorMessage',
         type: 'string',
-        href: 'exampleWithError',
+        description: `Used to communicate error information to the user. Be sure to localize the text. See the [states](#States) variant to learn more.`,
       },
       {
         name: 'helperText',
         type: 'string',
-        description: 'More information about how to complete the form field',
-        href: 'helperText',
+        description: `Used to provide more information about the form field. Be sure to localize the text. See the [helper text](#Helper-text) variant to learn more.`,
       },
       {
         name: 'id',
         type: 'string',
+        description:
+          'A unique identifier to connect the underlying `<select>` with the associated label',
         required: true,
-        href: 'basicExample',
       },
       {
         name: 'label',
         type: 'string',
+        description: 'The label shown above the input. Be sure to localize the label.',
       },
       {
         name: 'name',
         type: 'string',
-        href: 'basicExample',
+        description: 'Used to specify the name of the control.',
       },
       {
         name: 'onChange',
-        type: '({ event: SyntheticInputEvent<>, value: string }) => void',
+        type: '({| event: SyntheticInputEvent<>, value: string |}) => void',
+        description: 'Callback triggered when the user makes any changes',
```

Comment:
`any changes` seems like there are many options, but it's just `selects a new option`. We can't delete the option nor search for it.

Reply:
Sounds good - thanks!